docs/SUMMARY.md

```diff
@@ -14,10 +14,18 @@
   - [Sync modes]()
   - [Pruning]()
 
-- [Layer 2]()
-  - [Quickstart]()
+- [Layer 2](./l2/README.md)
+  - [General overview](./l2/overview.md)
+  - [Sequencer](./l2/sequencer.md)
+  - [Contracts](./l2/contracts.md)
+  - [Prover](./l2/prover.md)
+  - [State Diffs](./l2/state_diffs.md)
+  - [Withdrawals](./l2/withdrawals.md)
+  - [Aligned mode](./l2/aligned_mode.md)
```

Comment:
I'd review other L2s documentations and try to reorg this. We can do this in this PR or in another if we file an issue. I'd suggest to reviewing the following and others:
- https://docs.taiko.xyz/start-here/getting-started
- https://docs.zksync.io
- https://docs.optimism.io/stack/getting-started

Reply:
Yes. I was thinking of just moving the docs in this PR, to reduce potential conflicts (already happened with the prover docs 😬).

I'll create an issue for this

Reply:
I was thinking of only moving the docs in this PR, to minimize conflicts (already had some in the prover docs).

I opened an issue to follow this: https://github.com/lambdaclass/ethrex/issues/3174